docs/source/quickstart.rst

```diff
@@ -26,6 +26,23 @@ At the end of these short instructions, you will be able to run the following sn
         sleep(250) # wait 250 msec
         pixels_off() # turn off the GiggleBot eyes
         sleep(250) # wait 250 msec
+
+
+.. note::
+
+    There are 2 MCUs sitting on the BBC micro:bit:
+
+    * One MCU has the role of flashing the HEX file that gets copied over to the other MCU.
+    * And the second MCU houses the actual scripts and MicroPython firmware that you interact with (in the real world or at the terminal).
+
+    Both of these MCUs have their own firmware, but sometimes the firmware sitting on the MCU that hosts the scripts and the MicroPython firmware gets to be called a *runtime*. Other
+    times it's just called for what it is - a firmware, just like in the following examples:
+    
+    * `Exhibit A <https://microbit-micropython.readthedocs.io/en/v1.0.1/devguide/flashfirmware.html?highlight=firmware#>`_ from BBC micro:bit MicroPython docs.
+    * `Exhibit B <https://microbit-micropython.readthedocs.io/en/v1.0.1/devguide/hexformat.html>`_ from BBC micro:bit MicroPython docs.
+
+    So in our documentation, when we talk about the firmware, we are referring to the MicroPython firmware. And when we want to refer to the other MCU's firmware,
```

Comment:
No, not simple. When people jump directly to a link they will have missed this little bit, and will be confused.

Reply:
When a task is difficult, it's better to give to the other one the feeling that things are simple. If they find themselves in a difficult situation, they'll think that someone told them things are simple, so they'll justify to themselves they are the ones doing something wrong. And this feedback can be the precursor to doing something about it, which is the key to finishing the task. Not to mention that it acts as an encouragement.

Otherwise, if they don't get any sort of encouragement or assesment of how difficult the task is, they can wander a lot of time without doing anything and/or getting frustrated about it. And let's not even imagine what the opposite of this could do: telling them it's hard.

It has been proven countless of times that regardless of the task's complexity, it's just better to encourage people by telling the task at hand is easy.

Reply:
It is exactly the opposite. If you fail at an easy task then you get all discouraged, give up and walk away. Not everyone is motivated by a challenge. It is better to  succeed when told it's a hard task than to fail at what was said to be an easy task.  We've had this discussion before.  We want users to feel empowered, not challenged beyond their capacities. 

A better approach is to say something like
_ The next part is a bit challenging but by following these instructions you will get through it. _